data/Makefile

```diff
@@ -45,7 +45,7 @@ format:
 	npm run format:json
 
 export_food_format: export_food format
-	
```

Comment:
so it appears this file has been formatted despite `data` being blacklisted

Reply:
Data is blacklisted for ruff only. Here it's the end of file formatter who removed the trailing line